.github/workflows/host-image-ci.yml

```diff
@@ -46,12 +46,12 @@ jobs:
     uses: ./.github/workflows/reusable-build.yaml
 
   build-metapackage-deb-job:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
nit: We could avoid repeating this using a [variable](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables. Apparently those variables can be at [repo level ans shared between workflows](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-configuration-variables-for-multiple-workflows).

Reply:
@jemoreira does this look good?

Variable values here: https://github.com/google/android-cuttlefish/settings/variables/actions

Reply:
These kept giving me workflow parsing errors. Let's revisit this later.